analysis/go.mod

```diff
@@ -3,9 +3,28 @@ module github.com/ossf/package-feeds/analysis
 go 1.15
 
 require (
+	cloud.google.com/go v0.79.0 // indirect
+	cloud.google.com/go/storage v1.14.0 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/google/wire v0.5.0 // indirect
 	github.com/googleapis/gnostic v0.5.4 // indirect
+	github.com/gophercloud/gophercloud v0.1.0 // indirect
 	gocloud.dev v0.22.0
-	k8s.io/apimachinery v0.18.12
-	k8s.io/client-go v0.18.12
+	golang.org/x/crypto v0.0.0-20210314154223-e6e6c4f2bb5b // indirect
+	golang.org/x/mod v0.4.2 // indirect
+	golang.org/x/oauth2 v0.0.0-20210313182246-cd4f82c27b84 // indirect
+	golang.org/x/sys v0.0.0-20210314195730-07df6a141424 // indirect
+	golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d // indirect
+	golang.org/x/time v0.0.0-20210220033141-f8bda1e9f3ba // indirect
+	google.golang.org/genproto v0.0.0-20210312152112-fc591d9ea70f // indirect
+	gopkg.in/yaml.v2 v2.4.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	k8s.io/apimachinery v0.20.4
+	k8s.io/client-go v0.20.4
```

Comment:
client-go is always picky and I have had issues in the past when upgrading versions. Have you been able to test this?

Reply:
This was already running over the weekend and it seems to work fine.

Reply:
Cool! Thanks for the update.